config/default_records/descriptors/002_element_aviti_descriptors.wip.yml

```diff
@@ -40,52 +40,59 @@ PhiX Expiry:
   kind: Date
   required: false
   sorter: 5
+PhiX Concentration (pM):
+  name: PhiX Concentration (pM)
+  task: Loading
+  workflow: Element Aviti
+  kind: Date
+  required: false
+  sorter: 6
 PhiX %:
   name: PhiX %
   task: Loading
   workflow: Element Aviti
   kind: Text
   required: false
-  sorter: 6
+  sorter: 7
```

Comment:
Changing attributes of the existing records may require a custom deployment step to change them in the database. Please include them to deployment notes if there are custom steps and post-deployment checks in the story.